components/LanguageAndSpecialty.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <v-row>
+    <v-col>
+      <v-select
+        :items="items"
```

Comment:
I guess this is more of a UX design question, but do you think we should move the language option into the top navbar?

Im not sure if they would search for a different language 🤔 ....unless it's multiple languages....like English or Spanish but they don't know Japanese. 

I'm rambling, but maybe this should be a multiple choice drop down?
We could also default it to the page level language.